### PR TITLE
[Docker] Add custom secret names

### LINKED ISSUE #5349: [Feature Request] Custom MINIO_*_KEY_FILE location
Hi,

it would be useful to have `MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` as docker environment variables to allow mutiple keys and minio setups on one host.

## Expected Behavior
`MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` should be changeable through docker environment variables (like mysql or postgresql docker containers).

## Current Behavior
Currently `MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` are hard-coded in dockerscripts/docker-entrypoint.sh.

## Possible Solution
I think the best way is to rename `MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` in dockerscripts/docker-entrypoint.sh and use them as docker environment variables in Dockerfile so the user can deploy his setup with
```
environment:
- MINIO_ACCESS_KEY_FILE: "my_access_key"
- MINIO_SECRET_KEY_FILE: "my_secret_key"
```
Of course `/run/secrets` is fixed but the filenames could be variable.

I can create a PULL REQUEST later. I just want to clarify this before and would like to hear your opinion.

## Steps to Reproduce (for bugs)
No bug, just a feature request

## Context
I want to run multiple minio servers on one docker host using docker secrets. 

Thank you very much!

Best regards
Paul

### PR DESCRIPTION
**Pull request by ptrunk**
## Description
`MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` are now changeable through docker environment variables.

## Motivation and Context
Resolves #5349

## How Has This Been Tested?
Build and run successfully with custom key file paths 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by balamurugana** [comment]
```
mint-xl.sh: PASSED
---
mint-fs.sh: PASSED
---
mint-dist-xl.sh: PASSED
---

```

File: dockerscripts/docker-entrypoint.sh [hunk 1]
```diff
@@ -24,15 +24,15 @@ fi
...
-        if [ -f $MINIO_ACCESS_KEY_FILE ]; then
-            export MINIO_ACCESS_KEY="$(cat "$MINIO_ACCESS_KEY_FILE")"
+    if [ -f $ACCESS_KEY_FILE -a -f $SECRET_KEY_FILE ]; then
+        if [ -f $ACCESS_KEY_FILE ]; then
+            export MINIO_ACCESS_KEY="$(cat "$ACCESS_KEY_FILE")"
         fi
-        if [ -f $MINIO_SECRET_KEY_FILE ]; then
-            export MINIO_SECRET_KEY="$(cat "$MINIO_SECRET_KEY_FILE")"
+        if [ -f $SECRET_KEY_FILE ]; then
+            export MINIO_SECRET_KEY="$(cat "$SECRET_KEY_FILE")"
```
Comment (donatello) [inline dockerscripts/docker-entrypoint.sh:35]: The code here can be simplified. `[ -f $ACCESS_KEY_FILE -a -f $SECRET_KEY_FILE ]` already checks if both files exist, so the repeated checks inside the if block can be removed.

Comment (ptrunk) [reply dockerscripts/docker-entrypoint.sh:35]: Yes you're right! I've also noticed this. Thank you. But i didn't want to change too much in a single pull request. So i left the code style as it was to make a review easier.

**Comment by kannappanr** [comment]
ping @harshavardhana @nitisht

**Review by harshavardhana — APPROVED** [review]
LGTM

**Review by donatello — APPROVED** [review]

**Review by nitisht — APPROVED** [review]

**Pull request merged**